scripts/report.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+#  2025 NVIDIA CORPORATION & AFFILIATES
+#
+#  Licensed under the Apache License, Version 2.0 (the License);
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an AS IS BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+# This script will generate a sos report and a dpf report
+
+# Variables:
+# CASE_ID - The case number for the support case (optional)
+# DEBUG - Be more verbose (optional)
+
+set -eo pipefail
+
+OUTPUT_DIR=/host/tmp
+
+if [ -n "$CASE_ID" ]; then
+	case_id="--case-id $CASE_ID"
+fi
+if [ "$DEBUG" == "true" ] ; then
+	verbose="-v"
+fi
+
+options="$case_id $verbose"
+sos report -s /host $options | tee /tmp/log.txt
```

Comment:
nit: we could use a list to simplify the code, see

```
bash-5.2$ declare options
bash-5.2$ echo ${options[@]}

bash-5.2$ options+=(-v)
bash-5.2$ echo ${options[@]}
-v
bash-5.2$ options+=("--case-id xx")
bash-5.2$ echo ${options[@]}
-v --case-id xx
bash-5.2$
```

```suggestion
declare options
if [ -n "$CASE_ID" ]; then
	options+=("--case-id $CASE_ID")
fi
if [ "$DEBUG" == "true" ] ; then
	options+=("-v")
fi

sos report -s /host ${options[@]} | tee /tmp/log.txt
```